src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -213,8 +213,18 @@ def _load_package_tool(tool_name, module_name, class_name, method_name, node_inp
                 message=f"Required inputs {list(missing_inputs)} are not provided for tool '{tool_name}'.",
                 target=ErrorTarget.EXECUTOR,
             )
+        try:
+            api = getattr(provider_class(**init_inputs_values), method_name)
+        except Exception as ex:
+            error_type_and_message = f"({ex.__class__.__name__}) {ex}"
+            raise ToolLoadError(
+                module=module_name,
+                message_format=("Failed to load package tool '{tool_name}': {error_type_and_message}"),
```

Comment:
Let's remove the `()` here if the message is a single line.

Reply:
Done.